src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -2567,8 +2567,16 @@ public void Equals_Encyclopaedia_ReturnsExpected(StringComparison comparison, bo
         {
             string source = "encyclop\u00e6dia";
             string target = "encyclopaedia";
+            ThreadCultureChange culture;
+            if (PlatformDetection.IsBrowser)
+            {
+                culture = new ThreadCultureChange("pl-PL");
+            } else
+            {
+                culture = new ThreadCultureChange("se-SE");
+            }
 
-            using (new ThreadCultureChange("se-SE"))
```

Comment:
nit: this change looks ok. you can just write it as 

 using (new ThreadCultureChange(PlatformDetection.IsBrowser ?"pl-PL" : "se-SE"))